src/components/Image.ts

```diff
@@ -23,6 +29,7 @@ export type ImageChild = never;
  */
 export type ImageProps = {
 	data: Uint8Array | Promise<Uint8Array>;
+	type: FileMime | Promise<FileMime>;
```

Comment:
Is `type` necessarily a prop if it can be inferred from `data` anyway? Would prefer not to have it, because it makes the user-facing API a little more complex and its value might contradict `data`

Reply:
ok, i'll change this